catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -337,6 +344,12 @@
                 false /* tokenized */,
                 true /* multivalued */,
                 STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(Validation.VALIDATORS,
+                true /* indexed */,
```

Comment:
I'm implementing it with two separate lists as @pklinef suggests. If that's the case, would we still care for it being queryable, @harrison-tarr?